### PR TITLE
One-touch deploy: Set host to fineractmysql when running in docker

### PR DESCRIPTION
**Pull request by conradsp**
## Description
This is a quick update to the One-touch deploy process. When we seed the tenant database, we want to default the hostname to fineractmysql rather than localhost. Eventually, this will be all that we need to do, rather than updating the build.gradle file. 

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Commit message starts with the issue number from https://issues.apache.org/jira/projects/FINERACT/. Ex: FINERACT-646 Pockets API.

- [ ] Coding conventions at https://cwiki.apache.org/confluence/display/FINERACT/Coding+Conventions have been followed.

- [ ] API documentation at https://github.com/apache/fineract/blob/develop/api-docs/apiLive.htm has been updated with details of any API changes.

- [ ] Integration tests have been created/updated for verifying the changes made.

- [ ] All Integrations tests are passing with the new commits.

- [ ] Submission is not a "code dump".  (Large changes can be made "in repository" via a branch.  Ask on the list.)

Our guidelines for code reviews is at https://cwiki.apache.org/confluence/display/FINERACT/Code+Review+Guide

### CODE REVIEW

File: Dockerfile
Comment (vorburger) [inline Dockerfile:29]: > Eventually, this will be all that we need to do, rather than updating the build.gradle file.

only after https://issues.apache.org/jira/browse/FINERACT-765, or could this be done now already?

**Comment by vorburger** [comment]
@conradsp what bothers me slightly (but not enough to not merge this, after we clarify the small question, above, so this just FYI) is that as we get this on Docker Hub (https://issues.apache.org/jira/browse/INFRA-18673), we're going to be pushing a container image with a hard-ocded DB URL that is specific to `docker compose` .. I'd like to eventually contribute e.g. Kubernetes deployment steps, where the DB would be differerent. I can't really think of a better solution for this problem right now either though.

**Comment by conradsp** [comment]
Thanks Michael - I agree with your comments. As soon as
https://issues.apache.org/jira/browse/FINERACT-765 is merged, I will remove
the build.gradle changes in the Dockerfile.

Will continue to think about a more robust solution. Ideally the db seeding
scripts will not have a db hostname encoded at all and would pull that from
an environment variable (defaulting to localhost or some sane default).

Thanks,
Steve

**Comment by vorburger** [comment]
> As soon as FINERACT-765 is merged, I will remove the build.gradle changes in the Dockerfile.

OK, so for now we do need both (that wasn't clear to me; I still don't really fully understand it).

> Will continue to think about a more robust solution. Ideally the db seeding scripts will not have a db hostname encoded at all and would pull that from an environment variable (defaulting to localhost or some sane default).

+1 (I've created just https://issues.apache.org/jira/browse/FINERACT-773 re. this, for some day.)

**Review by vorburger — APPROVED** [review]

**Comment by vorburger** [comment]
see #613

**Pull request merged**
